Sum-of-numbers/getSum.js

```diff
@@ -0,0 +1,5 @@
+const getSum = (a, b) => {
+  let min = Math.min(a, b);
+  let max = Math.max(a, b);
```

Comment:
If you don't change value of these variables take it to `const` instead of `let`. Try do use `let` **only** if you have to.